metadata/lang/es.yml

```diff
@@ -0,0 +1,44 @@
+# metadata box
+overview: "Descripción General"
+questions: "Preguntas"
+objectives: "Objectivos"
+requirements: "TOTRANSLATE"
+time-estimation: "Duración estimada"
+minutes: "minutos"
+level: "Nivel"
+introductory: "Introductorio"
+intermediate: "TOTRANSLATE"
+advanced: "TOTRANSLATE"
```

Comment:
I've left a couple of TODOs for you in this file @galaxyproject/training-spanish team :)

Reply:
On it!